### ISSUE TITLE
[BUG] Emoji's and icons not showing.

### ISSUE THREAD

**Issue by programmer-boi** [elkowar/eww#381]
### Checklist before submitting an issue

- [X] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [X] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [X] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

### Description of the bug

When I launch my eww widgets, none of the icons or emojis show up.
I had to replace the icons with actual text so that I would know what is what, but obviously this looks ugly. I am aware that this is most likely not a bug with eww itself, but I've scoured the internet trying to find a solution and this is my last resort. Please no hate, I'm a very new user to Linux. Just started using it for a couple of weeks.

### Reproducing the issue

I'm using this guy's config. I made sure to use the yuck version of it to work with the latest version of eww. 
https://github.com/adi1090x/widgets
https://github.com/adi1090x/widgets/pull/5  --> (Updated to yuck config).

![image](https://user-images.githubusercontent.com/68497216/148803190-7677acbc-145e-4709-ae28-bd146eb8b22f.png)
As you could spot here instead of the icons there are these black boxes. When I copied these black boxes into google search i found out that some of these are actually invalid unicodes. I couldn't find a font that would support them.

![2022-01-04-161916_1920x1080_scrot](https://user-images.githubusercontent.com/68497216/148803268-b668a0e6-328e-457f-8902-5a494e8bd4ae.png)
Style is a bit different as I was trying to adjust to my theme.

### Expected behaviour

![image](https://user-images.githubusercontent.com/68497216/148803484-765edebe-3b29-4cfb-bbd8-c939525d32ef.png)
Basically this.

### Additional context

System info:
OS: Ubuntu 20.04.3 (Minimal Install) <--- Could this be an issue?
WM: xmonad
Compositor: Compton
Terminal: Kitty
Bar: xmobar

**Comment by programmer-boi** [comment]
I didn't have correct fonts installed... sorry for the bother.
